services/QuillLMS/spec/models/translated_text_spec.rb

```diff
@@ -51,4 +52,33 @@
       end
     end
   end
+
+  describe '#text' do
+    subject { translated_text.text }
+
+    let(:translated_text) { create(:translated_text)}
+
+    it 'is the text of the english text' do
+      expect(subject).to eq(translated_text.english_text.text)
+    end
+  end
+
+  describe '#source' do
+    subject { translated_text.source }
+
+    let(:translated_text) { create(:translated_text)}
+
+    context 'translation mapping exists' do
+      it 'is the source of the first translation_mapping' do
+        translation_mapping = create(:translation_mapping, english_text: translated_text.english_text)
```

Comment:
Small stylistic nit-pick that you can take or leave: you could move this `create` call into a `let!` declaration in the containing context.